packages/sdk/.npmignore

```diff
@@ -1,2 +1,53 @@
-demo/
-website/
\ No newline at end of file
+config/
```

Comment:
A suggestion: There is a way to manage the npm ignore patterns which ignores everything and use `!` to escape the essential folders. For example, https://github.com/microsoft/rushstack/blob/main/apps/rush/.npmignore